### DISCUSSION TITLE
Change working Frame work reactjs -> nextjs

### DISCUSSION THREAD

**Discussion by bellaabdelouahab** [ISIL-ESTE/Student-Workflow-Organizer#27] [Ideas]

**Comment by Khalid1G** [comment]
I would like to use tailwind CSS instead of native CSS to style our components, also It a best practice to use storybook for each component

-- what is storybook: https://storybook.js.org/docs/react/get-started/whats-a-story

**Comment by bellaabdelouahab** [comment]
> I would like to use tailwind CSS instead of native CSS to style our components, also It a best practice to use storybook for each component
> 
> -- what is storybook: https://storybook.js.org/docs/react/get-started/whats-a-story

We can still utilize CSS for complex animations if needed. As for Storybook, using it independently is fine. However, integrating it into the full project's frontend might not be suitable for everyone on the team. It could potentially slow down development if not everyone is familiar with it or if it introduces additional complexities.
@Khalid1G

**Comment by bellaabdelouahab** [comment]
@bouslama-hamza has already changed the framework from React to Next.js. I don't recommend this type of action, but for now, while the project is still in its early stage, it can be acceptable. However, after this change, we shouldn't accept any contributions unless they are related to the assigned issue.

this issue will be closed, but we can keep the the discussion about storybook
